test/parallel/test-fs-rm.js

```diff
@@ -280,3 +280,24 @@ function removeAsync(dir) {
     message: /^The value of "options\.maxRetries" is out of range\./
   });
 }
+
+{
+  // Check that deleting a file that cannot be accessed using rmsync throws
+  const dirname = nextDirPath();
+  try {
+    fs.mkdirSync(dirname, { recursive: true });
+    fs.writeFileSync(path.join(dirname, 'text.txt'), 'hello');
+    fs.chmodSync(dirname, '500');
+    assert.throws(() => {
+      fs.rmSync(`${dirname}/text.txt`, { force: true });
+    }, {
+      code: 'EACCES',
+      syscall: 'unlink',
+      name: 'Error',
+      errno: -13,
```

Comment:
```suggestion
```
`errno` can hold different values on different platforms, so I think we should just stick to `code`.